coriolis/utils.py

```diff
@@ -724,6 +724,36 @@ def create_service(ssh, cmdline, svcname, run_as=None, start=True):
             "could not determine init system")
 
 
+@retry_on_error()
+def restart_service(ssh, svcname):
+    if test_ssh_path(ssh, "/lib/systemd/system"):
+        exec_ssh_cmd(ssh, "sudo systemctl restart %s" % svcname, get_pty=True)
+    elif test_ssh_path(ssh, "/etc/init"):
+        exec_ssh_cmd(ssh, "restart %s" % svcname)
+    else:
+        raise exception.CoriolisException("could not determine init system")
```

Comment:
Could you please define a proper `UnrecognizedWorkerInitSystem` exception in the `coriolis.exceptions` module (likely inheriting `NotSupportedOperation`, but you can just inherit from the base `CoriolisException` too if needed) and give it a properly-capitalized default error message.